specutils/io/default_loaders/jwst_reader.py

```diff
@@ -192,6 +219,67 @@ def jwst_x1d_multi_loader(file_obj, **kwargs):
     return _jwst_spec1d_loader(file_obj, extname='EXTRACT1D', **kwargs)
 
 
+@data_loader("JWST x1d MIRI MRS", identifier=identify_jwst_miri_mrs,
+             dtype=SpectrumList, extensions=['*'])
+def jwst_x1d_miri_mrs_loader(input, missing="raise", **kwargs):
+    """
+    Loader for JWST x1d MIRI MRS spectral data in FITS format
+
+    A single data set consists of a bunch of _x1d files corresponding to
+    a variety of wavelength bands. This reader reads one by one and packs
+    the result into a SpectrumList instance.
+
+    Parameters
+    ----------
+    input : list with str or file-like
+        List of FITS file names, or objects (provided from name by
+        Astropy I/O Registry). Alternatively, a directory path on
+        which glob.glob runs with pattern an implicit pattern "_x1d.fits",
+        or a directory path with a glob pattern already set.
+    missing : str
+        Allows the user to continue loading if one file is missing by setting
+        the value to "warn" or "silent". In the first case a warning will be issued
+        to the user, in the latter the file will silently be skipped. Any other
+        value will result in a FileNotFoundError if any files in the list are missing.
+
+    Returns
+    -------
+    SpectrumList
+        A list of the spectra that are contained in all the files.
+    """
+    # If input is a list, go read each file. If directory, glob-expand
+    # list of file names.
+    if not isinstance(input, (list, tuple)):
+        if os.path.isdir(input):
+            file_list = glob.glob(os.path.join(input, "*_x1d.fits"), recursive=True)
+        else:
+            file_list = glob.glob(input, recursive=True)
+    else:
+        file_list = input
+
+    spectra = []
+    for file_obj in file_list:
+        try:
+            sp =  _jwst_spec1d_loader(file_obj, **kwargs)
+        except FileNotFoundError as e:
+            if missing.lower() == "warn":
```

Comment:
I don't see why we need to allow for mixed-case values. This is not FITS header.

```suggestion
            if missing == "warn":
```

Reply:
I just figured it's the most user-friendly way to handle string matching *shrug*